Assets/VRTK/Scripts/Locomotion/VRTK_SlingshotJump.cs

```diff
@@ -0,0 +1,283 @@
+    }
+
+    /// <summary>
+    /// Event Payload
+    /// </summary>
+    /// <param name="sender">this object</param>
+    /// <param name="e"><see cref="PlayerClimbEventArgs"/></param>
+    public delegate void SlingshotJumpEventHandler(object sender, SlingshotJumpEventArgs e);
+
+    /// <summary>
+    /// Slingshot Jump allows player jumping based on the direction and amount pulled back of each controller. This slingshots the player in defined direction and speed.
+    /// </summary>
+    /// <example>
+    /// `VRTK/Examples/037_CameraRig_ClimbingFalling` shows how to set up a scene with slingshot jumping. This script just needs to be added to the PlayArea object and the requested forces and buttons set.
+    /// </example>
+    [AddComponentMenu("VRTK/Scripts/Locomotion/VRTK_SlingshotJump")]
+    [RequireComponent(typeof(VRTK_BodyPhysics))]
+    public class VRTK_SlingshotJump : MonoBehaviour
+    {
+        [Tooltip("This button will cancel an already tensioned sling shot.")]
+        public VRTK_ControllerEvents.ButtonAlias cancelButton = VRTK_ControllerEvents.ButtonAlias.Undefined;
+        [Tooltip("This button will start the sling shot move.")]
+        public VRTK_ControllerEvents.ButtonAlias triggerButton =  VRTK_ControllerEvents.ButtonAlias.TriggerPress;
+        [Tooltip("How close together the trigger releases have to be to initiate a jump.")]
+        public float releaseWindowTime = 0.5f;
+        [Tooltip("Multiplier that increases the jump strength.")]
+        public float velocityMultiplier = 5.0f;
+        [Tooltip("The maximum velocity a jump can be.")]
+        public float velocityMax = 8.0f;
+
+        /// <summary>
+        /// Emitted when a slingshot jump occurs
+        /// </summary>
+        public event SlingshotJumpEventHandler SlingshotJumped;
+
+        protected Transform playArea;
+        protected VRTK_BodyPhysics bodyPhysics;
+        protected VRTK_PlayerClimb playerClimb;
+
+        protected Vector3 leftStartAimPosition;
+        protected Vector3 leftReleasePosition;
+        protected bool leftIsAiming;
+
+        protected Vector3 rightStartAimPosition;
+        protected Vector3 rightReleasePosition;
+        protected bool rightIsAiming;
+
+        protected VRTK_ControllerEvents leftControllerEvents;
+        protected VRTK_ControllerEvents rightControllerEvents;
+
+        protected VRTK_InteractGrab leftControllerGrab;
+        protected VRTK_InteractGrab rightControllerGrab;
+
+        protected bool leftTriggerReleased;
+        protected bool rightTriggerReleased;
+        protected float countDownEndTime;
+
+        protected virtual void Awake()
+        {
+            bodyPhysics = GetComponent<VRTK_BodyPhysics>();
+            playerClimb = GetComponent<VRTK_PlayerClimb>();
+
+            VRTK_SDKManager.instance.AddBehaviourToToggleOnLoadedSetupChange(this);
+        }
+
+        protected virtual void OnEnable()
+        {
+            InitListeners(true);
```

Comment:
I added a SetActivationButton() and SetCancelButton() function that can be called from code that will reset the button listeners.